### PR TITLE
Feature/contact

### PR DESCRIPTION
**Pull request by enzofab91**
### **Feature:**
As a user I should be able to contact TargetMVD administrators so I can ask them any questions I have.

### **Trello reference:**
https://trello.com/c/haUW7QDs

### **Description:**
This PR adds support for contacting MVD administrators.

### CODE REVIEW

File: app/controllers/api/v1/contacts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class ContactsController < Api::V1::ApiController
+      skip_before_action :authenticate_user!
+
+      def create
+        contact = Contact.create!(contacts_params)
+        AdminMailer.new_contact_question(contact)
```
Comment (mrodriguez90) [inline app/controllers/api/v1/contacts_controller.rb:8]: Maybe we can move this to the Contact model and call it on an after_create callback.

File: spec/requests/api/v1/contacts/create_spec.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+describe 'POST api/v1/contacts', type: :request do
+  subject { post api_v1_contacts_path, params: params, as: :json }
+
+  describe 'POST create' do
+    let(:email) { Faker::Internet.email }
+    let(:body) { Faker::Lorem.sentence }
+    let(:contact) { Contact.last }
```
Comment (mrodriguez90) [inline spec/requests/api/v1/contacts/create_spec.rb:7]: Do we need this?

**Review by mrodriguez90 — COMMENTED** [review]
💥

File: spec/requests/api/v1/contacts/create_spec.rb (hunk 2 above)
Comment (enzofab91) [reply spec/requests/api/v1/contacts/create_spec.rb:7]: Yes, those are the params sent to the request.

File: app/controllers/api/v1/contacts_controller.rb (hunk 1 above)
Comment (mrodriguez90) [reply app/controllers/api/v1/contacts_controller.rb:8]: It depends on the requirement, if we need a new_contact_question email to be sent every time we create a contact then the callback makes sense. But if we need the email to be sent only from this endpoint then the callback is not necessary.
Feel free to decide what's best, I don't how big is going to be the scope of this project/target.

**Review by mrodriguez90 — APPROVED** [review]

File: app/models/contact.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Contact < ApplicationRecord
+  validates :body, :email, presence: true
```
Comment (sebastiancaraballo) [inline app/models/contact.rb:2]: Annotate models

File: app/views/admin_mailer/new_contact_question.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<p> New question from <%= @contact.email %> </p>
```
Comment (sebastiancaraballo) [inline app/views/admin_mailer/new_contact_question.html.erb:1]: This should be i18n as well.

File: app/mailers/admin_mailer.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class AdminMailer < ApplicationMailer
+  def new_contact_question(contact)
+    from = 'no_reply@target.com'
```
Comment (sebastiancaraballo) [inline app/mailers/admin_mailer.rb:3]: This email should be a constant.

File: spec/factories/about.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :about do
```
Comment (sebastiancaraballo) [inline spec/factories/about.rb:2]: Factories should be annotated as well 🤔

**Review by sebastiancaraballo — APPROVED** [review]

**Pull request merged**